src/shim/telemetryClient.ts

```diff
@@ -307,7 +517,7 @@ export class TelemetryClient {
             contextObjects?: { [name: string]: any }
         ) => boolean
     ) {
-        throw new Error("Not implemented");
+        Logger.getInstance().warn("addTelemetryProcessor is not supported via the ApplicationInsights shim. Please upgrade to the ApplicationInsights SDK beta.");
```

Comment:
Incorrect message here, AI Shim is the same as AI SDK beta right now, telemetry processor are not supported anymore